src/System.Private.ServiceModel/tests/System.Private.ServiceModel.Tests.csproj

```diff
@@ -14,7 +14,7 @@
     <ProjectGuid>{BB3B114D-68D7-4E36-9EBE-127E694E6EC9}</ProjectGuid>
   </PropertyGroup>
   <ItemGroup>
-    <Compile Include="**\*.cs" Exclude="Scenarios\SelfHostWcfService\*" />
+    <Compile Include="**\*.cs" Exclude="Scenarios\**\*" />
```

Comment:
What does it mean to exclude "Scenarios\**\*"? It looks to me like it means to exclude any files which don't have an extension. As the include only pulls in things with a .cs extension, it would seem to me that this exclude would be a no-op?